libsolidity/ast/Types.cpp

```diff
@@ -3090,6 +3090,16 @@ string FunctionType::canonicalName() const
 	return "function";
 }
 
+string FunctionType::humanReadableName() const
+{
+	switch (m_kind)
+	{
+		case Kind::Error: return "error " + m_declaration->name();
```

Comment:
I think this should also include the signature like `toString` does, shouldn't it?

Reply:
As mentioned by @cameel  in the Solidity Dev chat, It's about function parameters
For eg. 
`function abc(uint x, bytes calldata b) public view {}`
The signature is `abc(uint,bytes)`, not just `abc`
Since errors are similar to function types, the same goes for them. Their signatures work the same way.